common/src/main/java/com/wynntils/features/user/ChatTimestampFeature.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.StartDisabled;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@StartDisabled
+public class ChatTimestampFeature extends UserFeature {
+    @Config
+    public String formatPattern = "HH:mm:ss";
+
+    @SubscribeEvent
+    public void onChat(ChatMessageReceivedEvent event) {
```

Comment:
ChatMessageReceivedEvent is from ChatModel. Make sure to add ChatModel to this feature's model dependencies.